widgetssdk/src/main/java/com/glia/widgets/core/visitor/Authentication.kt

```diff
@@ -0,0 +1,26 @@
+package com.glia.widgets.core.visitor
+
+import com.glia.androidsdk.visitor.Authentication
+
+/**
+ * Interface for managing authentication and de-authentication.
+ *
+ * The `Behavior` enum defines behavior for authentication and de-authentication
+ * in different scenarios.
+ */
+interface Authentication: Authentication {
```

Comment:
Do we need to inherit from the core's `Authentication` here?

Reply:
I think so. By doing this, we ensure that interfaces are aligned, Widgets don't miss any public Authentication functionality from Core. Also, it avoids some coding in other places.

What concerns/suggestions do you have in this matter?

Reply:
Makes sense!
I think this is a little bit confusiong `interface Authentication: Authentication`, maybe it makes sense to introduce import alias for core's `Authentication` smth like `CoreAuthentication`?

Reply:
I like this idea. 👍 
Changed dc0d45db1d19cfb57bf23bf62a979c0954818dfb.